signer/tests/integration/emily.rs

```diff
@@ -604,22 +603,23 @@ async fn get_deposit_request_works() {
     assert!(request.is_none());
 }
 
-#[cfg_attr(not(feature = "integration-tests"), ignore)]
-#[tokio::test]
+#[ignore = "Test tries to use an emily api call it cannot access."]
+// #[cfg_attr(not(feature = "integration-tests"), ignore)]
+// #[tokio::test]
 async fn get_limits_works() {
     let url = Url::parse("http://testApiKey@localhost:3031").unwrap();
     let emily_client = EmilyClient::try_from(&url).unwrap();
-    emily_client::apis::limits_api::set_limits(
-        &emily_client.config(),
-        Limits {
-            peg_cap: Some(Some(100)),
-            per_deposit_cap: Some(Some(90)),
-            per_withdrawal_cap: Some(Some(80)),
-            ..Default::default()
-        },
-    )
-    .await
-    .unwrap();
+    // emily_client::apis::limits_api::set_limits(
+    //     &emily_client.config(),
+    //     Limits {
+    //         peg_cap: Some(Some(100)),
+    //         per_deposit_cap: Some(Some(90)),
+    //         per_withdrawal_cap: Some(Some(80)),
+    //         ..Default::default()
+    //     },
+    // )
+    // .await
+    // .unwrap();
```

Comment:
nit: should this test also use the testing_emily_client or the private one?

Reply:
It can set it with the testing one, but the actual signer will still use the structures from the public one. If you'd rather I can have this pull in the test api client and then create the limits and use the other for the rest of the test

Reply:
This is postponed for a separete PR